### DISCUSSION TITLE
How to exclude property from BeEquivalentTo() globally

### DISCUSSION THREAD

**Discussion by LucasRufo** [fluentassertions/fluentassertions#2552] [Q&A] [answered]
Hello! 

I have a scenario in which I want to exclude a property when using BeEquivalentTo() globally. Is it possible? 

Right now I'm calling the Excluding every time that I have to verify this Type.

```csharp
object.Should().BeEquivalentTo(expectedObject, ctx => ctx.Excluding(p => p.Property));
```

Thank you!

**Comment by dennisdoomen** [comment, marked as answer]
Yes, the practical all non-generic options you pass to `BeEquivalentTo` can also be passed to `AssertOptions.AssertEquivalencyUsing`
